model/models/qwen25vl/model_vision.go

```diff
@@ -332,11 +332,9 @@ func (m *VisionModel) PositionalEmbedding(ctx ml.Context, grid *Grid) ml.Tensor
 	pos := ctx.Input().FromIntSlice(coords, 2, grid.Width, grid.Height)
 
 	// Reshape and permute positions to match spatial merging pattern
-	pos = pos.Reshape(ctx, 2, grid.Width, merge, grid.Height/merge)
+	pos = pos.Reshape(ctx, -1, grid.Width/merge, merge, grid.Height/merge)
```

Comment:
I'm not 100% sure what's going on here... I'm assuming the extra reshape just gets cut down here but does the tensor get reshaped to? The last part just converts it into a vector?